packages/regl-worldview/src/Worldview.js

```diff
@@ -165,6 +176,16 @@ export class WorldviewBase extends React.Component<BaseProps, State> {
   };
 
   _onMouseUp = (e: MouseEvent) => {
+    const { dragStartPos } = this.state;
+    if (dragStartPos) {
+      const deltaX = e.clientX - dragStartPos.x;
+      const deltaY = e.clientY - dragStartPos.y;
+      const distance = Math.sqrt(deltaX * deltaX + deltaY * deltaY);
+      if (distance > this.props.mouseClickRadius) {
+        this._disableOnClick = true;
+      }
+      this.setState({ dragStartPos: null });
+    }
```

Comment:
If you want to simplify things a bit, you can even just call `onClick` here (if the mouse has not moved beyond the radius). Then you can get rid of `_disableOnClick`.

Reply:
Good suggestion. Updated